packages/gatsby-theme-docs/src/layouts/content.js

```diff
@@ -53,87 +58,89 @@ const LayoutContent = props => {
     }
   `);
   return (
-    <SiteDataContext.Provider value={data.site}>
-      <Reset />
-      <Globals />
-      <LayoutApplication isMenuOpen={isMenuOpen}>
-        <LayoutHeader siteTitle={data.site.siteMetadata.title} />
-        <LayoutSidebar
-          isMenuOpen={isMenuOpen}
-          setMenuOpen={setMenuOpen}
-          slug={props.pageContext.slug}
-          siteTitle={data.site.siteMetadata.title}
-        />
-        <LayoutMain
-          css={css`
-            grid-column: 2;
-            grid-row: 2;
+    <ErrorBoundary>
```

Comment:
I put this here for now, we can then move it into more specific areas and have multiple of them.